src/domain/common/whiteboard/excalidraw/collab/Collab.ts

```diff
@@ -491,6 +445,10 @@ class Collab {
     return this.excalidrawAPI.getSceneElementsIncludingDeleted();
   };
 
+  public getFiles = () => {
```

Comment:
right, this is because previously portal was accessing directly some of these functions. I have changed the visibility of some functions at the end of this file